src/main.rs

```diff
@@ -18,7 +19,25 @@ async fn main() -> Result<(), Box<dyn std::error::Error>>
                     .default_value(OsStr::from(chrono::Utc::now().year().to_string())),
             ),
         )
-        .subcommand(clap::command!("run").args([Arg::new("day").short('d').required(false)]));
+        .subcommand(clap::command!("run").args([Arg::new("day").short('d').required(false)]))
+        .subcommand(
+            Command::new("token")
+                .about("Get or set the session token used to communicate with the AOC servers")
+                .arg_required_else_help(true)
+                .args([
+                    Arg::new("set")
+                        .short('s')
+                        .long("set")
+                        .exclusive(true)
+                        .help("Set the session token"),
+                    Arg::new("get")
+                        .short('g')
+                        .long("get")
+                        .exclusive(true)
+                        .num_args(0)
+                        .help("Print the current session token, if any"),
```

Comment:
:heart_eyes: We should add more help messages for the other arguments in a new PR ofc

Reply:
Yeah